test/e2e/e2e_test.go

```diff
@@ -153,3 +255,238 @@ var _ = Describe("e2e control plane", func() {
 		framework.ExpectNoError(<-errChan)
 	})
 })
+
+// Test e2e hybrid sdn inter-node connectivity between worker nodes
+var _ = Describe("test e2e hybrid sdn inter-node connectivity between specified nodes", func() {
+	var haMode bool
+	svcname := "internode-hyb-sdn-e2e"
+	pingTarget := "172.17.0.250"
+	ovnNs := "ovn-kubernetes"
+	ovnWorkerNode := "ovn-worker"
+	ovnWorkerNode2 := "ovn-worker2"
+	ovnHaWorkerNode := "ovn-control-plane2"
+	ovnHaWorkerNode2 := "ovn-control-plane3"
+	ovnContainer := "ovnkube-node"
+	ovnNsFlag := fmt.Sprintf("--namespace=%s", ovnNs)
+	f := framework.NewDefaultFramework(svcname)
+
+	// Determine what KIND environment is running by querying the running nodes
+	BeforeEach(func() {
+		labelFlag := fmt.Sprintf("name=%s", ovnContainer)
+		jsonFlag := "-o=jsonpath='{.items..metadata.name}'"
+		fieldSelectorFlag := fmt.Sprintf("--field-selector=spec.nodeName=%s", ovnWorkerNode)
+		fieldSelectorHaFlag := fmt.Sprintf("--field-selector=spec.nodeName=%s", ovnHaWorkerNode)
+		annotationFlag := fmt.Sprintf("k8s.ovn.org/hybrid-overlay-external-gw=%s", pingTarget)
+		// Annotate the namespace to place the pods into the hybrid-sdn bridge br-ext
```

Comment:
@nerdalert sorry I'm a bit lost here. So do we now have support for hybrid overlay in our CI? How does this work? Do we create a br-ext bridge on a node/host and set up a dummy vxlan tunnel?

Reply:
@trozet the design I saw was to setup a macvlan interface and treat it as the external interface. I am looking at how to do it with a dummy interface acting as a TEP endpoint but that is in a separate PR https://github.com/ovn-org/ovn-kubernetes/pull/1251 that will be rebased and finalized with the "Add external gw feature to hybrid overlay" PR #1209.

I have the namespace annotation in this PR, since it seems to be the trigger to setup `br-ext` bridge that will put subsequently created pods in that namespace and on the VXLAN bridge to verify inter-node connectivity. So while this test isn't actually testing a dummy endpoint, the annotation lets me test inside of the `br-ext` (vxlan) bridge and hybrid-sdn namespace which was broken last week in the PR. Ty.

Reply:
@nerdalert OK let's sync up tmrw if you have time and discuss this more. I'd like to understand how we are doing the hybrid overlay and how we can enable it in CI. I would like that stuff turned when we merge the tests.

Reply:
From our sync up we determined best path is:

1. we can merge this PR before external gw support is added
2. need to update enabling the gw support in CI
3. need to finish the external gw patch